lib/game_statistics.rb

```diff
@@ -0,0 +1,123 @@
+class GameStatistics
+  attr_reader :stat_tracker
+
+  def initialize(stat_tracker)
+    @stat_tracker = stat_tracker
+  end
+
+  def sum_team_scores
+    grouping = {}
+    total_score_data_set.map do |array|
+      if grouping[array[0]].nil?
```

Comment:
At a point in the future when we refactor, I want to see if we can get the position references out of our code. Going to note where these are for that time in the future.